app/controllers/restaurants_controller.rb

```diff
@@ -31,6 +32,10 @@ def create
 
   private
 
+  def find_restaurant
+    Restaurant.find(params[:id])
```

Comment:
this doesn't set an instance variable of @restaurant so it doesn't make sense.  This needs to be taken out.  You can just keep the file as it was before.